sqlalchemy_imageattach/entity.py

```diff
@@ -127,8 +127,16 @@ def image_attachment(*args, **kwargs):
        implementation.
 
     """
+    if 'has_many' in kwargs:
```

Comment:
`uselist` would be better for the parameter name.  There’s the same parameter name, which shares similar semantic, for SQLAlchemy’s `relationship()`.